base/operators.jl

```diff
@@ -328,9 +328,31 @@ eltype(::Type{Any}) = Any
 eltype(t::DataType) = eltype(supertype(t))
 eltype(x) = eltype(typeof(x))
 
-# function pipelining
+# function pipelining, composition & negation
+
 |>(x, f) = f(x)
 
+"""
+    ∘(f, g)
+    f ∘ g
```

Comment:
Yes, no need to explicitly ping me if I've already commented on this issue. It may be a unicode issue where you might have to explicitly declare unicode operators somewhere for RST/Sphinx (I'm not sure where that is though), or it may have to do with the separate signatures being on multiple lines.